src/components/timecard/start-finish-time-input/StartFinishTimeInput.jsx

```diff
@@ -29,22 +30,34 @@ const StartFinishTimeInput = ({
     setErrorMessages(relevantErrorMessages);
   };
 
+  const sortErrors = () => {
```

Comment:
Will all other areas of the ui need to follow this same logic for sorting errors? 
If so maybe it would be worth pulling out into a more generic util function or something that takes an array of the error keys and then call it anywhere that you need to sort the errors.

Reply:
I've moved this function now to a separate class